src/main/java/net/mcreator/ui/modgui/BlockGUI.java

```diff
@@ -191,7 +191,7 @@
 	private final JComboBox<String> particleSpawningShape = new JComboBox<>(
 			new String[] { "Spread", "Top", "Tube", "Plane" });
 
-	private final JSpinner particleSpawningRadious = new JSpinner(new SpinnerNumberModel(0.5, 0, 2, 0.1f));
+	private final JSpinner particleSpawningRadious = new JSpinner(new SpinnerNumberModel(0.5, 0, 100, 0.1));

```

Comment:
Why is this value now 100? If scale changed, updater is needed for existing workspaces

Reply:
There was no real reason to limit it at 2, and it doesn't affect existing workspaces since values above 2 weren't allowed anyway

Reply:
Sorry, my bad, did not see it was max, not default value :)